application/frontend/controllers/MfaController.php

```diff
@@ -246,4 +286,53 @@ public function actionUpdate(int $id)
 
         return $mfa;
     }
+
+
+    /**
+     * @param int $mfaId
+     * @param int $webauthnId
+     * @return MfaWebauthn
+     * @throws BadRequestHttpException
+     * @throws NotFoundHttpException
+     * @throws ServerErrorHttpException
+     * @throws \Throwable
+     * @throws \yii\db\StaleObjectException
+     */
+    public function actionUpdateWebauthn(int $mfaId, int $webauthnId)
+    {
+        $this->getRequestedMfa($mfaId);
```

Comment:
Since the return value of this isn't used, it looks (at first glance) like this line might be unnecessary. It appears (when I look closer) that it's validating that the MFA records exists and has the given Employee ID.

Would you mind adding a comment to communicate those side benefits of calling this method, since it's not obvious from the function name itself?